drafts/docker-open-liberty-raspberry-pi.adoc

```diff
@@ -1,60 +1,73 @@
-an image for the ARM Raspberry Pi.
+The complication when moving to a Raspberry Pi is that it has an ARM CPU at its heart, meaning that machine code compiled for x86 or x86-64 CPUs (the standard CPUs you get in laptops/desktops today) won’t execute. Liberty is written in Java so it doesn’t have to worry about what the CPU architecture is: as long as there is a Java runtime it’s able to run (as the Java runtime itself interprets the Java code and executes it on the hardware). We do have official Open Liberty images on Docker Hub, but they are layered on top of the official IBM image, which is built on top of Ubuntu and doesn’t have a JVM that will run on ARM.
 
-=== Open Liberty Images
-To start, take a copy of the link:https://github.com/OpenLiberty/ci.docker/tree/master/official/latest/kernel/java11/openj9[kernel Dockerfile for the x86 Liberty Docker image]. This can be used as the basis for custom built images that contain only the features required for a specific application. 
+This means you can’t take our Open Liberty Docker image from Docker Hub, which is designed for an x86 architecture, and run it on the Pi without any configuration to the Docker images. We need to edit the Dockerfiles that define how to build the x86 Liberty Docker image and modify them to build an image for the ARM Raspberry Pi.
 
-Edit the `FROM` target in the Dockerfile to base the Open Liberty image on an link:https://hub.docker.com/r/arm32v7/adoptopenjdk/[official OpenJDK] image that is compatible with ARM.
+=== Open Liberty images
+To start, take a copy of the kernel Dockerfile for the link:https://github.com/OpenLiberty/ci.docker/tree/master/official/latest/kernel/java12/openj9[x86 Liberty Docker image]. This can be used as the basis for custom built images that contain only the features required for a specific application. 
+Edit the `FROM` target in the Dockerfile to base the Open Liberty image on an link:https://hub.docker.com/r/arm32v7/adoptopenjdk/[OpenJDK] image that is compatible with ARM. For example: 
 
 [source]
-FROM arm32v7/adoptopenjdk:11-jre-hotspot
+----
+FROM arm32v7/adoptopenjdk:latest
 ...
 CMD ["/opt/ol/wlp/bin/server", "run", "defaultServer"]
+----
 
-Build this image so it is stored locally. You will now be able to use this image as a base for your applications. Instructions to build a Docker image that has your application and configuration pre-loaded can be found link:https://github.com/OpenLiberty/ci.docker#building-an-application-image[here]. 
+Download the other folders and files in that directory to build this image so it is stored locally. You will now be able to use this image as a base for your applications. Instructions to build a Docker image that has your application and configuration pre-loaded can be found link:https://github.com/OpenLiberty/ci.docker#building-an-application-image[here]. 
```

Comment:
"the other folders and files in that directory" - Which directory? Do you mean they should clone the ci.docker repo? In which case, instead of just saying to take a copy of the kernel Dockerfile, just tell them to clone the repo from the start.

Reply:
"You can now use this image as a base for your application." (use present, not future tense)

Reply:
Don't use "here" as a link - it needs to make sense in isolation. eg "Follow the instructions to build a Docker image with your application and configuration pre-loaded in the URL[ci.docker README]." (that's probably not the best way to put it so if you have a better way, feel free.) The other change I made there was to make the sentence more 'active' - instead of telling them where they can find the instructions, just tell them to follow them - it's less ambiguous about what you want them to do.